src/main/java/org/apache/accumulo/proxy/ProxyServer.java

```diff
@@ -2091,8 +2092,9 @@ private AuthenticationToken getToken(String principal, Map<String,String> proper
     props.putAllStrings(properties);
     AuthenticationToken token;
     try {
-      token = tokenClass.newInstance();
-    } catch (InstantiationException | IllegalAccessException e) {
+      token = tokenClass.getDeclaredConstructor().newInstance();
+    } catch (InstantiationException | IllegalAccessException | NoSuchMethodException
+        | InvocationTargetException e) {
```

Comment:
I think you can just do:

```suggestion
    } catch (ReflectiveOperationException e) {
```